src/resources/refund.js

```diff
@@ -0,0 +1,44 @@
+import T from 'proptypes';
+import base from './base';
+
+export const propTypes = {
+  id: T.string,
+  tracking_code: T.string,
+  confirmation_number: T.string,
+  status: T.string,
+  carrier: T.string,
+  shipment_id: T.string,
+};
+
+export default (api) =>
+  class Refund extends base(api) {
+    static _name = 'Refund';
+
+    static _url = 'refunds';
+
+    static key = 'refund';
+
+    static propTypes = propTypes;
+
+    /**
+     * create a refund
+     * @returns {Promise<never>}
+     */
+    static async save(params = {}) {
```

Comment:
Do we need a custom overridden function here to save or can we use the `base` save function instead?

Reply:
The `base` save function is non-static and we need to have a wrappedParams for the `refund` parameter.